Makefile

```diff
@@ -202,7 +211,75 @@ deploy-artefact:
 	echo TODO: $(@)
 
 apply-data-changes:
-	echo TODO: $(@)
+	eval "$$(make aws-assume-role-export-variables)"
+	http_result=$$(aws lambda invoke --function-name $(PROJECT_ID)-$(PROFILE)-service-etl out.json --log-type Tail | jq .StatusCode)
```

Comment:
Just double check that the functions follow this naming convention. If they dont, they need to be updated to be this.

Reply:
Yeah, pretty sure the name is right.